examples/next/app/api/chat/route.ts

```diff
@@ -55,10 +55,21 @@ export async function POST(req: Request) {
   // save the user message
   saveChat({ id, messages, activeStreamId: null });
 
+  const userStopSignal = new AbortController();
+
   const result = streamText({
     model: 'openai/gpt-5-mini',
     messages: convertToModelMessages(messages),
-    abortSignal: req.signal,
+    abortSignal: userStopSignal.signal,
```

Comment:
this is the relevant chane. Before, the stream to the provider was aborted when the user navigated away. Now, the stream is only aborted when the user explicitly stops it.